Scanning for ServiceLoader fails with Java 10

When I enable the `addServiceUses` configuration, the plugin fails on Java 10/11 with this root cause  exception:

```
Caused by: java.lang.IllegalArgumentException
	at org.moditect.internal.shaded.asm.ClassReader.<init>(Unknown Source)
	at org.moditect.internal.shaded.asm.ClassReader.<init>(Unknown Source)
	at org.moditect.internal.shaded.asm.ClassReader.<init>(Unknown Source)
	at org.moditect.internal.analyzer.ServiceLoaderUseScanner.getUsedServices(ServiceLoaderUseScanner.java:60)
	at org.moditect.internal.analyzer.ServiceLoaderUseScanner.lambda$getUsedServices$1(ServiceLoaderUseScanner.java:47)
	at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.accept(ForEachOps.java:183)
	at java.base/java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:177)
	at java.base/java.util.zip.ZipFile$EntrySpliterator.tryAdvance(ZipFile.java:578)
	at java.base/java.util.Spliterator.forEachRemaining(Spliterator.java:326)
	at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:484)
	at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:474)
	at java.base/java.util.stream.ForEachOps$ForEachOp.evaluateSequential(ForEachOps.java:150)
	at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateSequential(ForEachOps.java:173)
	at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
	at java.base/java.util.stream.ReferencePipeline.forEach(ReferencePipeline.java:497)
	at org.moditect.internal.analyzer.ServiceLoaderUseScanner.getUsedServices(ServiceLoaderUseScanner.java:47)
	at org.moditect.commands.GenerateModuleInfo.updateModuleInfo(GenerateModuleInfo.java:262)
	at org.moditect.commands.GenerateModuleInfo.run(GenerateModuleInfo.java:201)
	at org.moditect.mavenplugin.generate.ModuleInfoGenerator.generateModuleInfo(ModuleInfoGenerator.java:131)
	at org.moditect.mavenplugin.add.AddModuleInfoMojo.getModuleInfoSource(AddModuleInfoMojo.java:310)
	at org.moditect.mavenplugin.add.AddModuleInfoMojo.execute(AddModuleInfoMojo.java:150)
	at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo(DefaultBuildPluginManager.java:134)
	... 21 more
```

Possibly a newer version of ASM should be shaded?

--- a/core/pom.xml
+++ b/core/pom.xml
@@ -38,7 +38,7 @@
         <dependency>
             <groupId>org.ow2.asm</groupId>
             <artifactId>asm</artifactId>
-            <version>6.0_BETA</version>
+            <version>6.2.1</version>
         </dependency>
         <dependency>
             <groupId>com.beust</groupId>

--- a/core/src/main/java/org/moditect/internal/compiler/ModuleInfoCompiler.java
+++ b/core/src/main/java/org/moditect/internal/compiler/ModuleInfoCompiler.java
@@ -21,7 +21,7 @@ import static org.objectweb.asm.Opcodes.ACC_OPEN;
 import static org.objectweb.asm.Opcodes.ACC_STATIC_PHASE;
 import static org.objectweb.asm.Opcodes.ACC_SYNTHETIC;
 import static org.objectweb.asm.Opcodes.ACC_TRANSITIVE;
-import static org.objectweb.asm.Opcodes.V1_9;
+import static org.objectweb.asm.Opcodes.V9;
 
 import java.io.IOException;
 import java.nio.file.Path;
@@ -74,7 +74,7 @@ public class ModuleInfoCompiler {
 
     public static byte[] compileModuleInfo(ModuleDeclaration module, String mainClass, String version) {
         ClassWriter classWriter = new ClassWriter( 0 );
-        classWriter.visit( V1_9, ACC_MODULE, "module-info", null, null, null );
+        classWriter.visit( V9, ACC_MODULE, "module-info", null, null, null );
 
         int moduleAccess = module.isOpen() ? ACC_SYNTHETIC | ACC_OPEN : ACC_SYNTHETIC;
         ModuleVisitor mv = classWriter.visitModule( module.getNameAsString(), moduleAccess, version );

